jdaviz/app.py

```diff
@@ -2048,15 +2047,25 @@ def _create_data_item(data):
             typ = 'cube'
         else:
             typ = 'unknown'
+
         # we'll expose any information we need here.  For "meta", not all entries are guaranteed
         # to be serializable, so we'll just send those that we need.
+
+        def _expose_meta(key):
```

Comment:
I don't understand the intent of this. I thought it is supposed to do the opposite of what you coded up, so a docstring to explain would be nice.

Reply:
docstring added, let me know if its not clear!